schema.graphql

```diff
@@ -460,10 +545,20 @@ type MatchQuestion {
   Information about the bonus that was read for this question. Must be omitted if `bonus_points` is present; may be omitted if no bonus was read for this question, but should not be omitted merely because no points were earned on the bonus.
   """
   bonus: MatchQuestionBonus
+
+  """
+  The time at which the moderator began reading the bonus. An ISO 8601-formatted string, with time zone, in the tournament's local time zone.
+  """
+  bonus_start_time: Date
+
+  """
+  Information about the bonus question used as a replacement in this cycle, if any. Omit if the bonus was not replaced. Multiple replacement bonuses are not currently supported.
+  """
+  replacement_bonus: MatchQuestionBonus
 }
 
 """
-Docs group: Match
+Docs group: Question
```

Comment:
Ditto for leaving this and the rest of the MatchQuestion* types in the Match group.

Reply:
Ditto